docs/source/prototype.rnnt.rst

```diff
@@ -0,0 +1,110 @@
+torchaudio.prototype.rnnt
+=========================
+
+.. py:module:: torchaudio.prototype
+
+.. currentmodule:: torchaudio.prototype
+
+Model Classes
+-------------
+
+Conformer
+~~~~~~~~~
+
+.. autoclass:: Conformer
+
+  .. automethod:: forward
+
+
+Emformer
+~~~~~~~~
+
+.. autoclass:: Emformer
+
+  .. automethod:: forward
+
+  .. automethod:: infer
```

Comment:
i would like to later split these out into a separate module (tentatively `torchaudio.prototype.nn`) to indicate that they are generic modules rather than models or rnnt-specific components

Reply:
Sure. I am open to re-organizing them.